apps/vanilla/src/pages/blackjack/index.css

```
@@ -0,0 +1,42 @@
+@import url("colors.css");
+
+.game-view {
+  width: 600px;
+  height: 800px;
+  margin: auto;
+}
+
+.debug-info {
+  border-bottom: 2px solid black;
+  border-top: 2px solid black;
+}
+
+.debug-info__name,
+.debug-info__name-score-separator,
+.debug-info__current-score {
+  margin: 0;
+  display: flex;
+  align-items: center;
+  text-align: center;
+}
+
+.debug-info__history-of-dice-values {
```

Try to make selectors simpler
```suggestion
.debug-info__dice-history {
```